witness/src/Dscp/Snowdrop/AccountValidation.hs

```diff
@@ -63,6 +65,13 @@ validateSimpleMoneyTransfer = mkValidator ty
   where
     ty = StateTxType $ getId (Proxy @TxIds) AccountTxTypeId
 
+data Authenticated payload = Authenticated
```

Comment:
Leave a comment please, this is going to be used all over the project iiuc

Reply:
Ok. Didn't know we want to spill snowdrop validtors all over the place, but hey.

Reply:
Right, I just saw a familiar pattern, but in other places we use `sign` :man_facepalming: